pkg/metrics/exporter.go

```diff
@@ -81,7 +84,35 @@ func (m *promMetricsExporter) Init() error {
 	view.RegisterExporter(m.ocExporter)
 
 	// start metrics server
-	return m.startMetricServer()
+	err = m.startMetricServer()
+	if err != nil {
+		return err
+	}
+
+	opts := m.exporter.Options()
+	// check if start pushgatway support
+	if !opts.pushGatewayEnabled {
```

Comment:
This change always uses the push gateway if it's enabled on the exporter, but we want that to be optional in Dapr too.

Please make changes that keep the existing behavior as is, and add an option (via the Configuration object) to use the push gateway only if it's enabled as a setting.

Reply:
ping @LaurenceLiZhixin